src/es/components/controllers/favorites/Favorites.js

```diff
@@ -116,4 +130,61 @@ export default class Favorites extends Shadow() {
       throw new Error(response.statusText)
     })
   }
+renderNotification(dependsElementName, description, position, renderingDuration = 4000, type = "success",) {
```

Comment:
A controller should never render anything or do anything directly in the dom. it is a non visual component.

Reply:
you can find this in the toolbox readme:

"Event Driven Rules:

    A controller must not query-select nor output any css or html. The communication must be strictly through events."